test/mocha.opts

```diff
@@ -1,4 +1,4 @@
 --require ./node_modules/ts-node/register
 --require ./node_modules/source-map-support/register
 --recursive
-test/*.ts
\ No newline at end of file
+test/**/*.ts
```

Comment:
nit: newline.

If you're using VS Code, you can punch these into your user settings:
```
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true,
    "files.trimTrailingWhitespace": true,
```

Reply:
https://github.com/xpring-eng/Xpring-JS/pull/112

Reply:
Added newline. Sent PR to put VSCode settings into project repo.